### ISSUE TITLE
Implement new endpoint for fetching generated designs

### ISSUE THREAD

**Issue by fushar** [Automattic/wp-calypso#63485]

**Comment by taipeicoder** [comment]
I believe this task depends on adding categories to recipes, correct? We should return one recipe per category.

**Comment by fushar** [comment]
> I believe this task depends on adding categories to recipes, correct? We should return one recipe per category.

Nope, for this task, I specifically mentioned this (and also check the AC):

> For now, we can implement an endpoint which does (1).

🙂
